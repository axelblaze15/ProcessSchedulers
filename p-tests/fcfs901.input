10000000000000 250 1
1000 8 1
1000000000 250 1
1000000 200 100
1000000000000000000 250 1
10 4 1