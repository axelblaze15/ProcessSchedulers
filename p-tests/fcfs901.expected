10 5 0 1
1000 9 0 1
1000000 100200 0 100000
1000000000 100201 98200 98201
10000000000000 100202 98201 98202
1000000000000000000 100203 98202 98203
49101 65768